$NetBSD: patch-CMakeLists.txt,v 1.1 2021/06/23 08:33:03 nia Exp $

Recognize "powerpc" as a valid variant of "ppc" so configure
works on NetBSD/macppc.

Maybe this should be normalized in CMake...

--- CMakeLists.txt.orig	2020-03-04 17:16:37.000000000 +0000
+++ CMakeLists.txt
@@ -71,11 +71,11 @@ elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR
     set(WTF_CPU_X86_64 1)
 elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "(i[3-6]86|x86)")
     set(WTF_CPU_X86 1)
-elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "ppc")
+elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc|powerpc)")
     set(WTF_CPU_PPC 1)
-elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64")
+elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc|powerpc)64")
     set(WTF_CPU_PPC64 1)
-elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
+elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc|powerpc)64le")
     set(WTF_CPU_PPC64LE 1)
 elseif (LOWERCASE_CMAKE_SYSTEM_PROCESSOR MATCHES "parisc*")
     set(WTF_CPU_HPPA 1)
